components/tdisplays3/display.py

```diff
@@ -78,6 +84,7 @@ async def to_code(config):
     cg.add_build_flag("-DTFT_D6=47")
     cg.add_build_flag("-DTFT_D7=48")
 
+    cg.add_build_flag("-DSPI_FREQUENCY=40000000")
```

Comment:
Is this mandatory? Can't it be done in the standard esphome way (i.e. configuring SPI)?

Reply:
Yeah perhaps it could. I just added it here because this whole component had all this settings defined here.

Reply:
For me it worked without specifying the frequency; that's why I'm asking whether it is mandatory.